app/assets/javascripts/organisation-list-filter.js

```diff
@@ -8,6 +8,8 @@
     init: function() {
       // Form should only appear if the JS is working
       $('[data-filter="form"]').addClass('filter-organisations-list__form--active');
+      var $results = $('<div/>').addClass('filter-organisations-list__results govuk-heading-m').attr('id', 'search-results').attr('aria-live', 'polite').html(window.GOVUK.filter.countInitialDepartments() + ' results found');
+      $('#organisations_search_results').prepend($results);
```

Comment:
Suggest using a `js-` class here instead of an id to more clearly show to future people that this is where JavaScript stuff happens in the template.